app/forms/pre_qualification_form.rb

```diff
@@ -1,15 +1,15 @@
 class PreQualificationForm
   include ActiveModel::Model
 
-  attr_accessor :question_1, :question_2, :question_3, :question_4, :question_5
+  attr_accessor :firm_active_question, :firm_business_model_question, :firm_status_question, :firm_particular_market_question
```

Comment:
I reckon we can drop the `firm_` prefix from all of these. Otherwise the naming is spot on.